### PR TITLE
CORCI-902 test: Only compress logs over 1Mb in size.

### PR DESCRIPTION
**Pull request by ashleypittman**
Change the bzip options so files under 1Mb in size uncompressed are not compressed.
Remove some legacy core file handling.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1140,11 +1140,12 @@ pipeline {
                                   rm -rf "Functional/"
                                   mkdir "Functional/"
                                   # compress those potentially huge DAOS logs
-                                  if daos_logs=$(ls install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/*); then
+                                  if daos_logs=$(find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size 2M); then
                                       lbzip2 $daos_logs
                                   fi
```
Comment (brianjmurrell) [inline Jenkinsfile:1145]: You could simplify this a bit but more importantly you want **+** before the `2M`:
```suggestion
                                  find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size +2M -print0 | 
                                      xargs -0r lbzip2
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1140,11 +1140,12 @@ pipeline {
...
                                   arts="$arts$(ls *daos{,_agent}.log* 2>/dev/null)" && arts="$arts"$'\n'
                                   arts="$arts$(ls -d install/lib/daos/TESTING/ftest/avocado/job-results/* 2>/dev/null)" && arts="$arts"$'\n'
+                                  arts="$arts$(ls install/lib/daos/TESTING/ftest/*.stacktrace 2>/dev/null || true)"
```
Comment (brianjmurrell) [inline Jenkinsfile:1148]: Why is this needed?  Stacktrace collection is done in `launch.py` so that stacktraces are filed with the test they occurred in.

File: Jenkinsfile (hunk 1 above)
Comment (ashleypittman) [reply Jenkinsfile:1145]: Ah. -size 2M catches files that are larger than 1M, +1M is probably the same but clearer as you say.

File: Jenkinsfile (hunk 2 above)
Comment (ashleypittman) [reply Jenkinsfile:1148]: Some sections had this already, some didn't so I was making it consistent, I don't really know why it was different, do you have any ideas?

File: Jenkinsfile (hunk 1 above)
Comment (brianjmurrell) [reply Jenkinsfile:1145]: I believe `-size 2M` catches files that are exactly 2M and no others.  None bigger or smaller.

File: Jenkinsfile (hunk 2 above)
Comment (brianjmurrell) [reply Jenkinsfile:1148]: Yeah, looks like #1886 probably was started before the multi-hardware-stage testing and didn't catch up when it merged.

So these should all be removed, not missing ones added.  Good catch noticing though.

And yes, it's not at all lost on us that this `Jenkinsfile` could use a lot of DRYing up.  John's got an experiment with matrix in `Jenkinsfile` that could achieve that.  Not sure I'm a fan of how it ends up running through.

File: Jenkinsfile (hunk 1 above)
Comment (ashleypittman) [reply Jenkinsfile]: I think it was probably doing it in buckets, so catching files between 2M and 3M in size..

File: Jenkinsfile [hunk 3]
```diff
@@ -1140,7 +1140,7 @@ pipeline {
                                   rm -rf "Functional/"
                                   mkdir "Functional/"
                                   # compress those potentially huge DAOS logs
-                                  if daos_logs=$(ls install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/*); then
+                                  if daos_logs=$(find install/lib/daos/TESTING/ftest/avocado/job-results/*/daos_logs/* -maxdepth 0 -type f -size +1M); then
```
Comment (brianjmurrell) [inline Jenkinsfile:1143]: While I agree that it's no worse than it was, I didn't realize how bad it was, but I wonder far away from an _Argument list too long_.  :open_mouth:   Here's an example from your test run:

```
[2020-04-03T12:43:14.685Z] ++ find install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.50-521b6c6-checksum-basic_checksum/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.52-f7fd4ed-container-basic_tx_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-76f16e6-container-simple_create_delete_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.53-8e44876-container-global_handle/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.54-d304282-container-snapshot/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.56-f4ef433-control-dmg_nvme_scan_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_agent.log-wolf-102vm6 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos.log-wolf-102vm6 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-59fb006-io-llnl_mpi4py/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T11.57-e2fe03b-control-super_block_versioning/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_agent.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.00-cf4b9a4-io-romio/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-30bdf55-object-punch_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.02-4f07882-object-array_obj_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.04-30729af-pool-attribute/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.10-b34ff6f-pool-connect_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-1634c07-pool-destroy_rebuild/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_agent.log-wolf-102vm8 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.11-5c39d70-pool-destroy_tests/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_admin.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_control.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.14-977dee4-pool-evict_test/daos_logs/daos_server.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-4a651d8-pool-info_tests/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.22-fdbedd1-pool-global_handle/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.31-8b83b48-pool-multiple_creates_test/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_admin.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_control.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm3 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm4 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.33-776ba4a-pool-pool_svc/daos_logs/daos_server.log-wolf-102vm5 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.34-087626f-pool-rebuild_tests/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.34-51bc187-pool-rebuild_no_cap/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-4c8a5cd-pool-rebuild_with_io/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-505a5a9-pool-rebuild_with_ior/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_agent.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.35-916d448-security-pool_connect_init/daos_logs/daos_server.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_admin.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_control.log-wolf-102vm2 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos.log-wolf-102vm1 install/lib/daos/TESTING/ftest/avocado/job-results/job-2020-04-03T12.37-8907023-server-daos_server_config/daos_logs/daos_server.log-wolf-102vm2 -maxdepth 0 -type f -size +1M
```

Not something that necessarily needs fixing here, but something for us to just keep in mind and now particularly that we are using `find` options to fix it are much more plentiful.

**Comment by jolivier23** [comment]
have you updated tickets for the test failures?

**Comment by ashleypittman** [comment]
> have you updated tickets for the test failures?

No, but I have verifed none of them relate to the change at hand.
